conda/install.py

```diff
@@ -369,16 +369,18 @@ def replace_long_shebang(mode, data):
 def replace_prefix(mode, data, placeholder, new_prefix):
     if mode == 'text':
         data = data.replace(placeholder.encode('utf-8'), new_prefix.encode('utf-8'))
-    elif mode == 'binary':
+    # Skip binary replacement in Windows.  Some files do have prefix information embedded, but
+    #    this should not matter, as it is not used for things like RPATH.
+    elif mode == 'binary' and sys.platform != "win32":
```

Comment:
We should use one of the five `on_win` functions here instead of `sys.platform` directly.